code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -203,21 +203,25 @@
 	QDEL_NULL(access_card)
 	return ..()
 
-/mob/living/simple_animal/bot/proc/check_access(mob/living/user)
+/mob/living/simple_animal/bot/proc/check_access(mob/living/user, obj/item/card/id)
 	if(user.has_unlimited_silicon_privilege || isAdminGhostAI(user)) // Silicon and Admins always have access.
 		return TRUE
 	if(!maints_access_required) // No requirements to access it.
 		return TRUE
 	if(!(bot_cover_flags & BOT_COVER_LOCKED)) // Unlocked.
 		return TRUE
 
-	var/obj/item/card/id/id_card = user.get_idcard(TRUE)
-	if(!id_card || !id_card.access)
+	var/obj/item/card/id/used_id
+	used_id = user.get_idcard(TRUE)
+	if(id) //IDs used directly overwrites
+		used_id = id
+
+	if(!used_id || !used_id.access)
 		return FALSE
-	id_card = id_card.GetID()
+	used_id = used_id.GetID()
```

Comment:
why does it even do this ❓

Reply:
The original intent was to work with PDA's/tablets but the code above already does so... oops.